client/src/components/Navbar.js

```diff
@@ -36,23 +37,33 @@ const SearchBar = styled.input`
 export const Navbar = () => {
   return (
     <nav className="nav">
-      <Title>Project Metroid</Title>
+      <Title className="title">Project Metroid</Title>
+      <button className="mobile-search-toggle nav__mobile--link">
+        <i class="fas fa-search"></i>
+      </button>
+      <button className="mobile-user-profile nav__mobile--link">
+        <i class="fas fa-user-circle"></i>
+      </button>
+      <button className="mobile-hamburger-menu nav__mobile--link">
+        <i class="fas fa-bars"></i>
+      </button>
       <div className="search-bar__container">
         <i class="search-bar__icon fas fa-search"></i>
         <SearchBar
           type="search"
           name="Search"
-          id="search-bar"
+          className="search-bar"
           placeholder="Search"
         />
       </div>
-
-      <a className="nav__link" href="#">
-        Discover
-      </a>
-      <a className="nav__link" href="#">
-        Browse
-      </a>
+      <div className="nav__link__container">
+        <a className="nav__link" href="#">
+          Discover
+        </a>
+        <a className="nav__link" href="#">
+          Browse
+        </a>
+      </div>
       <div className="login-links">
```

Comment:
className should be more BEM like `login__links-container` so that we know its a container, containing our login__links.